api/app/controllers/api/v1/applications_controller.rb

```diff
@@ -16,7 +16,14 @@ def index
 
         # POST /applications
         def create
+            # if we passed in an id that exists, we want to update
+            if params[:id] && Application.exists?(params[:id])
+                update and return
+            end
+
+            params.require(:applicant_id)
             if invalid_id(Session, :session_id) then return end
```

Comment:
I'm assuming that something like `:session_id` is passed in the JSON body right? If not, where did we store it?

Reply:
Yes, the `session_id` is stored in the JSON body.